Client/Frontend/WelcomeFlow/WelcomeFlowSignInView.swift

```diff
@@ -81,12 +81,22 @@ struct WelcomeFlowSignInView: View {
                         self.showError = true
                     }
                 ) {
-                    model.prevScreen = nil
+                    model.clearPreviousScreens()
                     model.changeScreenTo(.defaultBrowser)
                 }
             }
 
-            // TODO: sign-in with QR code
+            if FeatureFlag[.qrCodeSignIn] {
```

Comment:
To test drive this, you can simply make a change like this and then run the build.

```suggestion
            if FeatureFlag[.qrCodeSignIn] || true {
```